pkgs/applications/graphics/ahoviewer/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, pkgs, fetchurl, fetchFromGitHub, pkgconfig, libconfig, gtkmm, glibmm, libxml2, gstreamer, libsecret, curl, unrar, libzip, autoconf, automake }:
```

Comment:
You might want to wrap long lines as these as well to make navigation through it easier. Most people here still wrap at 80 characters. But this is nothing crucial.